Makefile

```diff
@@ -1,7 +1,7 @@
 TEST_VERSION := v1-3
-TEST_DATASET := NA12878-trio
+TEST_DATASET := na12878-trio
```

Comment:
Don't have a strong opinion, but just flagging for consideration:

- Do we want two digit versions? We haven't defined what minor vs major means; would it be easier to just have one number?
- For directory names, we usually use underscores, not hyphens (but we use hyphens for bucket and repository names).

Reply:
The second digit is more like a "patch". I use `TEST_VERSION` for quick tests only, so this number should only occur in the `tmp` bucket and not be exposed to `main`/`test` runs.